src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -137,6 +137,7 @@ public async Task SendStreamLimitRequestsConcurrently_Succeeds(int streamLimit)
                 {
                     await using Http3LoopbackStream stream = await connection.AcceptRequestStreamAsync();
                     await stream.HandleRequestAsync();
+                    _output.WriteLine($"[{DateTime.Now:T}] Server: Finished request {i}");
```

Comment:
```suggestion
                    _output.WriteLine($"[{DateTime.Now:HH:mm:ss.fffffff}] Server: Finished request {i}");
```
"T" gives only seconds, this will print the full granularity of the timestamp.